src/main/java/LunaBot.java

```diff
@@ -0,0 +1,245 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class LunaBot {
+
+    // Array of strings to store list of tasks
+    private static ArrayList<Task> taskList;
+    private static Storage storage;
+
+    public static void main(String[] args) {
+
+        storage = new Storage("./data/LunaBot.txt");
+
+        try {
+            taskList = storage.load();
+        }
+        catch (LunaBotException e) {
+            System.out.println("Unable to load tak list from file: " + e.getMessage());
+            taskList = new ArrayList<>();
+        }
+
+        // Greet the user
+        System.out.println("___________________________________________________________________");
+        System.out.println(" Hello! I'm LunaBot");
+        System.out.println(" What can I do for you?");
+        System.out.println("___________________________________________________________________");
+
+        // Create scanner for user input
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        // loop scanner to echo user input
+        while (true) {
+            try {
+                input = scanner.nextLine();
+                if (input.equals("bye")) {
```

Comment:
Perhaps you can use input.equalsIgnoreCase("bye") to handle cases where user might input "BYE"?